source/appModules/explorer.py

```diff
@@ -216,6 +226,13 @@ def chooseNVDAObjectOverlayClasses(self, obj, clsList):
 			clsList.insert(0, StartButton)
 			return # Optimization: return early to avoid comparing class names and roles that will never match.
 
+		if(
+			(winVersion.winVersion.major, winVersion.winVersion.minor) == (6, 1)

```

Comment:
I think I prefer the MetadataEditField overlay to be added to all these fields.
Having said that, have you come across cases wherhe RichEdit50W is used as the window class for edit boxes in Explorer that aren't metadata edit fields? How about edit fields in the properties window of files, for example?

Reply:
Are you suggesting to use editTextInfo for all RichEdits in Win Explorer regardless of the Windows version? I am personally against it because Windows Explorer is also used in Control Panel, and it isn't realistic to check all edit fields there under every version of Windows. I personally think that the check implemented at present might not be enough, however I haven't found any other negatively affected fields under Win 7. The main priority when bug fixing for me is not to introduce regression so better to be safe than sorry.